ppml/services/kms-utils/docker/README.md

```diff
@@ -55,9 +55,11 @@ INFO [main.cpp(159) -> main]: ehsm-kms enroll app end.
 
 
 export kms_type=ehsm_or_simple_or_azure
+export appid=your_appid
+export appkey=your_apikey
 
 # Generatekeys
-docker exec -i $ENROLL_CONTAINER_NAME bash -c "bash /home/entrypoint.sh $kms_type generatekeys"
```

Comment:
I haven't seen the following commands, but there should be no parameter $kms_type